docs/templating.md

```diff
@@ -972,7 +973,69 @@ a b c d e f
 ### dump (object)
```

Comment:
signature changed from `dump (object)` to `dump (object, [space])`, but I think in any case simply `dump` would be better as heading.